pkg/network/protocols/http/model_windows.go

```diff
@@ -84,65 +78,73 @@ func dstIPHigh(tup *driver.ConnTupleType) uint64 {
 //
 
 // ReqFragment returns a byte slice containing the first HTTPBufferSize bytes of the request
-func (tx *FullHttpTransaction) ReqFragment() []byte {
+func (tx *WinHttpTransaction) ReqFragment() []byte {
 	return tx.RequestFragment[:]
 }
 
-func (tx *FullHttpTransaction) StatusClass() int {
+func (tx *WinHttpTransaction) StatusClass() int {
 	return statusClass(tx.Txn.ResponseStatusCode)
 }
 
-func (tx *FullHttpTransaction) RequestLatency() float64 {
+func (tx *WinHttpTransaction) RequestLatency() float64 {
 	return requestLatency(tx.Txn.ResponseLastSeen, tx.Txn.RequestStarted)
 }
 
-func (tx *FullHttpTransaction) isIPV4() bool {
+func (tx *WinHttpTransaction) isIPV4() bool {
 	return isIPV4(&tx.Txn.Tup)
 }
 
-func (tx *FullHttpTransaction) SrcIPLow() uint64 {
+func (tx *WinHttpTransaction) SrcIPLow() uint64 {
 	return srcIPLow(&tx.Txn.Tup)
 }
 
-func (tx *FullHttpTransaction) SrcIPHigh() uint64 {
+func (tx *WinHttpTransaction) SrcIPHigh() uint64 {
 	return srcIPHigh(&tx.Txn.Tup)
 }
 
-func (tx *FullHttpTransaction) SrcPort() uint16 {
+func (tx *WinHttpTransaction) SrcPort() uint16 {
 	return tx.Txn.Tup.CliPort
 }
 
-func (tx *FullHttpTransaction) DstIPLow() uint64 {
+func (tx *WinHttpTransaction) DstIPLow() uint64 {
 	return dstIPLow(&tx.Txn.Tup)
 }
 
-func (tx *FullHttpTransaction) DstIPHigh() uint64 {
+func (tx *WinHttpTransaction) DstIPHigh() uint64 {
 	return dstIPHigh(&tx.Txn.Tup)
 }
 
-func (tx *FullHttpTransaction) DstPort() uint16 {
+func (tx *WinHttpTransaction) DstPort() uint16 {
 	return tx.Txn.Tup.SrvPort
 }
 
-func (tx *FullHttpTransaction) Method() Method {
+func (tx *WinHttpTransaction) Method() Method {
 	return Method(tx.Txn.RequestMethod)
 }
 
-func (tx *FullHttpTransaction) StatusCode() uint16 {
+func (tx *WinHttpTransaction) StatusCode() uint16 {
 	return tx.Txn.ResponseStatusCode
 }
 
 // Static Tags are not part of windows driver http transactions
-func (tx *FullHttpTransaction) StaticTags() uint64 {
+func (tx *WinHttpTransaction) StaticTags() uint64 {
 	return 0
 }
 
 // Dynamic Tags are not part of windows driver http transactions
```

Comment:
I think this comment is wrong now

Reply:
They're not part of driver transactions, they are part of ETW transactions